ansible-scylla-manager/tasks/main.yml

```diff
@@ -1,24 +1,4 @@
 ---
-- name: deploy local Scylla on the Manager node
-  import_role:
-    name: "{{ role_path }}/../ansible-scylla-node"
-  vars:
-  # TODO how to use scylla_manager_db_vars to be passed on as role params?
-    install_only: True
-    scylla_manager_enabled: false
-    scylla_version: 'latest'
-    scylla_edition: "{{ scylla_manager_db_vars.scylla_edition|default('oss') }}"
-    elrepo_kernel: false
```

Comment:
I get the impression that this manager setup wants the most basic local ScyllaDB setup as possible. Ideally, there shouldn't be much to pass in terms of variables in such a case.

This variable it already happens to be false by default in the scylla-node role. Since it's not doing anything, I removed it.

I.e. I think we should align the scylla-node role to have the most 'basic' defaults as possible. Minimal hassle to get an out-of-the-box working experience.

Reply:
I've placed the variable back, and we can address such variables and defaults in a separate PR.